GCS/probe0/probe0.proto

```diff
@@ -0,0 +1,19 @@
+syntax = "proto2";
+
+import "github.com/google/cloudprober/probes/proto/config.proto";
+import "github.com/googleinterns/step224-2020/config/proto/probe.proto";
+option go_package = "github.com/googleinterns/step224-2020/GCS/probe0/proto";
+
+//The probe0 package is the package for the test probe - creates files in GCS with contents without any significance
+
+package probe0;
+
+message ProbeConf {
+  required string bucket = 1;
+  required string object = 2;
+  optional hermes.HermesProbeDef probedef = 3;     
```

Comment:
Add some comments to better explain what is the purpose for each of these.

Reply:
Comments added